cmd/loop/liquidity.go

```diff
@@ -158,19 +158,165 @@ func setParam(ctx *cli.Context) error {
 		)
 	}
 
+	// Just set the rules on our current set of parameters and leave the
+	// other values untouched.
+	otherRules = append(otherRules, newRule)
+	params.Rules = otherRules
+
 	// Update our parameters to the existing set, plus our new rule.
 	_, err = client.SetLiquidityParams(
 		context.Background(),
 		&looprpc.SetLiquidityParamsRequest{
-			Parameters: &looprpc.LiquidityParameters{
-				Rules: append(otherRules, newRule),
-			},
+			Parameters: params,
+		},
+	)
+
+	return err
+}
+
+var setParamsCommand = cli.Command{
+	Name:        "setparams",
+	Usage:       "update the parameters set for the liquidity manager",
+	Description: "Updates the parameters set for the liquidity manager.",
+	Flags: []cli.Flag{
+		cli.Float64Flag{
+			Name: "maxswapfee",
+			Usage: "the maximum percentage of swap volume we are " +
+				"willing to pay in server fees, expressed as " +
+				"a decimal (eg, 0.01 = 1%)",
+		},
+		cli.Float64Flag{
+			Name: "maxroutingfee",
+			Usage: "the maximum percentage of off-chain payment " +
+				"volume that are are willing to pay in " +
+				"routing fees, expressed as a decimal (eg, " +
+				"0.01 = 1%).",
+		},
+		cli.Float64Flag{
+			Name: "maxprepayfee",
+			Usage: "the maximum percentage of off-chain prepay " +
+				"volume that are are willing to pay in " +
+				"routing fees, expressed as a decimal (eg, " +
+				"0.01 = 1%).",
+		},
+		cli.Uint64Flag{
+			Name: "maxprepay",
+			Usage: "the maximum no-show (prepay) in satoshis that " +
+				"swap suggestions should be limited to.",
+		},
+		cli.Uint64Flag{
+			Name: "maxminer",
+			Usage: "the maximum miner fee in satoshis that swap " +
+				"suggestions should be limited to.",
+		},
+		cli.IntFlag{
+			Name: "sweepconf",
+			Usage: "the number of blocks from htlc height that " +
+				"swap suggestion sweeps should target, used " +
+				"to estimate max miner fee.",
+		},
+		cli.Uint64Flag{
+			Name: "failurebackoff",
+			Usage: "the amount of time, in seconds, that " +
+				"should pass before a channel that " +
+				"previously had a failed swap will be " +
+				"included in suggestions.",
+		},
+	},
+	Action: setParams,
+}
+
+func setParams(ctx *cli.Context) error {
```

Comment:
the name of the command `getParams` isn't totally consistent, because it also returns the rules

Reply:
indeed, couldn't really think of a better name for `setParams`, alternatively we could separate `getParams` into `getParams` and `getRule` but I don't think that adds much?

Reply:
`getConfig`?

Reply:
thought about it, but it could be confused with the actual loop config?